src/discrete-layer/components/layer-details/field-value-presentors/lookup.options-presentor.tsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import { Box } from '@map-colonies/react-components';
+import { MenuItem, Select } from '@map-colonies/react-core';
+import React, { useContext } from 'react';
+import { useIntl } from 'react-intl';
+import TooltippedValue from '../../../../common/components/form/tooltipped.value';
+import lookupTablesContext, { ILookupOption, LookupKey } from '../../../../common/contexts/lookupTables.context';
+import { EntityFormikHandlers } from '../layer-datails-form';
+import { IRecordFieldInfo } from '../layer-details.field-info';
+
+interface LookupTablesPresentorProps {
+  lookupKey: LookupKey;
+  fieldInfo: IRecordFieldInfo;
+  value?: string;
+  formik?: EntityFormikHandlers;
+}
+
+export const LookupOptionsPresentorComponent: React.FC<LookupTablesPresentorProps> = ({ lookupKey, fieldInfo, value, formik }) => {
+  const intl = useIntl(); 
+  const { lookupTablesData } = useContext(lookupTablesContext);
+
+  if (!lookupTablesData) return <></>;
+  const lookupOptions = lookupTablesData[lookupKey] as ILookupOption[];
+
+  if (!formik) {
+    return (
+      <TooltippedValue className="detailsFieldValue">
+        {"tooltip test value"}
```

Comment:
Handle this condition differently.. without the hard coded value, take inspiration from enum.value-presentor.tsx